### DISCUSSION TITLE
docker database on restarting loop - Synology NAS

### DISCUSSION THREAD

**Discussion by PigiR** [teslamate-org/teslamate#3309] [General]
Hi,

after reinstalling Teslamate to the latest version, can't manage to bring it up due to the db on a restarting loop.

Any suggestion?

**Comment by cwanja** [comment]
Do you have a backup of the database? Have you tried dropping the database volume and re-initializing it? 

Follow step 2 [here](https://docs.teslamate.org/docs/maintenance/upgrading_postgres) and then restore your [database following these steps](https://docs.teslamate.org/docs/maintenance/backup_restore).

**Reply by PigiR** [reply]
Hi, fresh everything, so no backups involved.

**Reply by PigiR** [reply]
deleted the db volume and recreated. Fixed
